.travis.yml

```diff
@@ -23,6 +23,8 @@ before_script:
 script:
     # flake8 the diff
     - "git diff HEAD^ '*.py' | flake8 --diff --config flake8.cfg"
+    # add secret key to django settings.py
+    - echo "SECRET_KEY= '$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 100 | head -n 1)'" >> platform/pulpcore/app/settings.py
```

Comment:
This line ensures that travis can run `makemigrations` and `manage.py test` without Django complaining. 
I don't think having a bad secret for our travis testing is an issue, since I really doubt anyone would use this for their pulp environment and the travis instance does not live for very long.